config.ru

```diff
@@ -5,3 +5,4 @@
 require_relative "config/environment"
 
 run Rails.application
+Rails.application.load_server
```

Comment:
the rails app update would replace what we had with this, so I'm not sure if these statements could conflict or not. To review?